sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -492,12 +491,19 @@ def __init__(self, options, root_staging_location=None):
     self._root_staging_location = (
         root_staging_location or self.google_cloud_options.staging_location)
 
-    self.environment_version = _FNAPI_ENVIRONMENT_MAJOR_VERSION
+    from apache_beam.runners.dataflow.dataflow_runner import _is_runner_v2_disabled
+    from google.cloud import storage
+    if _is_runner_v2_disabled(options):
```

Comment:
Seems irrelevant change. Rebase error? Specifically it undo [this change](https://github.com/apache/beam/commit/ca674aad086e4d1a40e8c01598073030a22484fa#diff-e616be01292a173f978dab1922575bbe06c2554b3bc6073de4aa1117b81f5a13) on master

Reply:
It does. Not sure how that happened. If we're no longer supporting the legacy runner (as I seem to recall we are not), I'll make sure to change it.